src/outlet/ticketStorage.ts

```diff
@@ -402,4 +402,19 @@ export class TicketStorage {
 	private storeTickets() {
 		localStorage.setItem(TicketStorage.LOCAL_STORAGE_KEY, JSON.stringify(this.ticketCollections))
 	}
+
+	public migrateLegacyTokenStorage(issuerConfig: OffChainTokenConfig, tokenStorageKey = 'dcTokens') {
```

Comment:
@nicktaras You could probable just pass collectionId rather than the whole issuer config, which will already be defined.
Alternatively we could make it a static function that doesn't rely on any class properties.

Reply:
Sounds good to improve this.